### DISCUSSION TITLE
Get the tag message in a pipeline

### DISCUSSION THREAD

**Discussion by michael-brade** [woodpecker-ci/woodpecker#2825] [Request a Feature]
Hi, here is one last question for now (got everything else working :))

When creating releases with `woodpeckerci/plugin-gitea-release`, there is the setting `note`. Is it possible to somehow get the tag message into that? I had a look at https://woodpecker-ci.org/docs/usage/environment, but none of the available variables seem to have the tag message. So is there any other way?

**Comment by qwerty287** [comment]
That's not possible, yet, I'll convert this into a feature request
